aws_lambda_powertools/utilities/parser/types.py

```diff
@@ -1,13 +1,14 @@
 """Generics and other shared types used across parser"""
 
+import sys
 from typing import TypeVar
 
 from pydantic import BaseModel
 
 # We only need typing_extensions for python versions <3.8
-try:
+if sys.version_info >= (3, 8):
     from typing import Literal  # noqa: F401
-except ImportError:
+else:
```

Comment:
That's an interesting one, perhaps `noqa` might not be needed after all?

Reply:
@heitorlessa  that is for the unused imports in `types.py`

Reply:
@heitorlessa adding `if sys.version_info >= (3, 8):` is a recommendation from MyPy to aid the detection some how